doc/development.md

```diff
@@ -132,3 +132,29 @@ Before we can start you need to match some prerequirements on your host machine.
     and run it in a seperate terminal window as this process will stay in the foreground.
 
   7. Now you can access you crowbar development setup via ```http://your.crowbar.instance:5000```
+
+## Debugging
+
+If you want to debug some specific piece of code you can use `byebug` which is also mentioned in the `Gemfile`
+
+To set it up you have to install the package:
+
+```bash
+zypper ar -f http://dist.suse.de/install/SLP/SLE-12-SP2-SDK-LATEST/x86_64/DVD1/ sle12-sp2-sdk
```

Comment:
In the devsetup the byebug gem is installed via bundler and the SDK repo is added as well.